### ISSUE TITLE
Rule not detected for influence graph in toy-jim.bngl

### ISSUE THREAD

**Issue by GoogleCodeExporter** [RuleWorld/rulebender#44]
```
I was just thinking about toy-jim and I realized that there is a problem with 
the Influence Graph.  It is actually missing an influence that is tricky to 
detect.  The transphosphorylation reactions (Rules 5 and 6) actually require 
dimerization to take place, which requires Rule 2 to fire.  So, in principle 
there should be a solid green arrow from Rule 2 to each of these.  However, the 
way the rules have been written obscures this dependency and a more 
sophisticated analysis would be required to pick it up.  This highlights a 
problem of using shorthand in rules, in particular the '.' to make indirect 
connections between molecules.  On the other hand, writing out the rules in 
full here would be very tedious (because the patterns would have to trace 
K->A->R->R->A->K).  Not sure how to explain this concisely or what exactly it 
says about using RB.

- from Jim
```

Original issue reported on code.google.com by `AdamMatthewSmith` on 11 Apr 2011 at 3:07
